test/k8sT/Policies.go

```diff
@@ -430,22 +429,17 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 
 			By("Testing if egress policy enforcement is enabled on the endpoint")
```

Comment:
This `By` is actually incorrect; we should update it to say that we are ensuring that both ingress and egress policy are being enforced.